outputs/http_test.go

```diff
@@ -66,7 +66,7 @@ func TestHTTPClient_Send(t *testing.T) {
 		{
 			name:          "sad path method get - bad url",
 			method:        http.MethodGet,
-			expectedError: `Get "http://path-to-nowhere": dial tcp: lookup path-to-nowhere: no such host`,
+			expectedError: `Get "http://path-to-nowhere": dial tcp: lookup path-to-nowhere: Temporary failure in name resolution`,
```

Comment:
I noticed this as well and fixed it here https://github.com/aquasecurity/postee/pull/287/commits/cc6c7a5d8df1f3aa5e53740180947c540646d406